src/panel/TemplateEditor/lib/cloud/looker.ts

```diff
@@ -0,0 +1,8 @@
+
+import * as request from "./request";
+
+export const load = async (lookId: number, filters = {} as { [ key: string ]: unknown }) => {
+    const { url } = await request.json(`/pricespider/looker-embed-url?look=${lookId}&filters=${JSON.stringify(filters)}`);
```

Comment:
Make this distinction about looker and pricespider in the cloud, passing the template path and the template params, such that the editor has no knowledge of pricespider or looker. The user context is already defined in the auth, so if you need to know something about for the pricespider you can get that by querying the users collection on the server.